### PR TITLE
Update sample.md

### PR DESCRIPTION
**Pull request by cal97g**
Pythonic import + choice method

## Description

**Resolves** #(issue number) 

## What does your PR belong to?
- [ ] Website
- [x] Snippets
- [ ] General / Things regarding the repository (like CI Integration)

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (non-breaking improvement of a snippet)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. 
- [ ] I have checked that the changes are working properly
- [ ] I have checked that there isn't any PR doing the same
- [ ] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by cal97g** [comment]
@Chalarangelo why do we import things without their namespace? I thought it was generally accepted that in production codebases something like `namespace.method` is preferable to `method`.

I can update this PR to add a description for `random.choice`; I do think it is a more pythonic way of retrieving a random element from a list.

Thanks!

**Comment by Chalarangelo** [comment]
Things are imported like this across the board for consistency and brevity. Using `random.choice()` seems like a good idea, you can update the description if you like.

File: snippets/sample.md [hunk 1]
```diff
@@ -5,15 +5,15 @@ tags: list,random,beginner
 
 Returns a random element from a list.
 
-Use `random.randint()` to generate a random number that corresponds to an index in the list, return the element at that index.
+Use `random.choice()` to select at random an element from a list.
```
Comment (Chalarangelo) [inline snippets/sample.md]:
```suggestion
Use `random.choice()` to select at random an element from the list.
```

File: snippets/sample.md [hunk 2]
```diff
@@ -5,15 +5,15 @@ tags: list,random,beginner
...
+
 
-[`random.sample()`](https://docs.python.org/3/library/random.html#random.sample) provides similar functionality to this snippet.
```
Comment (Chalarangelo) [inline snippets/sample.md]: Restore this line, please.

File: snippets/sample.md [hunk 3]
```diff
@@ -5,15 +5,15 @@ tags: list,random,beginner
...
 
 ```py
-from random import randint
+from random import choice
 
-def sample(lst):
-  return lst[randint(0, len(lst) - 1)]
+def sample(items):
```
Comment (Chalarangelo) [inline snippets/sample.md]:
```suggestion
def sample(lst):
```

File: snippets/sample.md [hunk 4]
```diff
@@ -5,15 +5,15 @@ tags: list,random,beginner
...
+  return choice(items)
```
Comment (Chalarangelo) [inline snippets/sample.md]:
```suggestion
  return choice(lst)
```

**Comment by cal97g** [comment]
If you want the code to be character by character as you've suggested you can do it.

**Pull request closed without merging**